src/ui/App.js

```diff
@@ -8,12 +10,13 @@ import DrawButton from "./DrawButton.js";
    Better yet, let's change it to a different "Play Again" button that
    resets everything and plays another hand (with a new Deck).
 **/
-export default function App({ initialCards }) {
+export default function App({ initialCards, fetchedDeck }) {
   const [cards, setCards] = useState(initialCards);
   const [selected, setSelected] = useState([]);
   const [unselectedAcesCount, setUnselectedAcesCount] = useState(
     numberOfUnselectedAces(cards)
   );
+  const deck = new Deck(fetchedDeck.deckID);
```

Comment:
Ideally, you'd use some sort of static factory method inside the client-side Deck class that would encapsulate the deck id so you wouldn't need to fetch it. In fact, what is fetchedDeck anyway? Shouldn't that be a Deck object?